pact/coin-contract/v4/coin-v4.repl

```diff
@@ -0,0 +1,1049 @@
+(env-data
```

Comment:
let's get rid of this indirection (left over from using v2 repl in v3)

Reply:
Confirmed that this is coin v2 repl test (with gas fixes) and coin v3 repl test (with event fixes) 👍